CHANGELOG.md

```diff
@@ -46,6 +46,7 @@
 * [7754](https://github.com/grafana/loki/pull/7754) **ashwanthgoli** index-shipper: add support for multiple stores.
 * [8662](https://github.com/grafana/loki/pull/8662) **liguozhong**: LogQL: Introduce `distinct`
 * [9813](https://github.com/grafana/loki/pull/9813) **jeschkies**: Enable Protobuf encoding via content negotiation between querier and query frontend.
+* [9920](https://github.com/grafana/loki/pull/9920) **vlad-diachenko**: Bump dskit version.
```

Comment:
I don't think we should make a changelog entry for updating a dependency, except is has some user-facing changes.

Reply:
I also was thinking about it )) it makes sense

Reply:
make format fixed it